src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -552,6 +552,14 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
             ? this.tasksApiNew.updateTaskAssignment
             : this.updateTaskAssignment;
 
+        const updateTask = isFeatureEnabled(features, 'activityFeed.tasks.newApi')
```

Comment:
Feature strings have started to repeat. we should be making them constants as I think mentioned in a prior PR sometime back

Reply:
Since this is a path, we should try to come up with a flow type that can either validate the path based on the feature object or at least add the constant checks to the feature checking component (at the risk of restricting the paths further than is actually necessary). Otherwise we have several definitions for the allowed values that can get out of sync. I can look into that as a followup.